scripts/nym-node-setup/auto-bond/show_balances.py

```
@@ -1,113 +1,162 @@
-NC = "\033[0m"
+UNYM_PER_NYM     = 1_000_000
+AUX_DETAILS_PATH = "/api/v2/auxiliary-details"
+NODE_HTTP_PORT   = 8080
+
+LCD_ENDPOINTS = [
+    "https://api.nymtech.net",
+    "https://api.nyx.nodes.guru",
+]
+LCD_BALANCE_PATH = "/cosmos/bank/v1beta1/balances/{address}"
+
+G  = "\033[0;32m"; R = "\033[0;31m"; Y = "\033[0;33m"
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)